specification/storage/resource-manager/readme.md

```diff
@@ -26,7 +26,34 @@ These are the global settings for the Storage API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2019-06
+tag: package-2020-08-preview
```

Comment:
We might should keep the default tag as the latest GA API version.
As I get from other thread, some feature (like Access Time tracking) target to GA API version. So if GA API version swagger will still be updated, we will have GA SDK release for it. So we might should still keep the default tag as GA API version.